tools/eth-migration/fetch-control-keys.mjs

```diff
@@ -0,0 +1,98 @@
+// Only way to silence PolkadotJS API warnings we don't want
+console.warn = () => {};
+
+import { ApiPromise, WsProvider } from "@polkadot/api";
+
+const SOURCE_URL = "wss://1.rpc.frequency.xyz";
+const BATCH_SIZE = 1000;
+
+const options = {
+	rpc: {
+		msa: {
+			getKeysByMsaId: {
+				description: 'Fetch Keys for an MSA Id',
+				params: [
+					{
+						name: 'msa_id',
+						type: 'MessageSourceId',
+					},
+				],
+				type: 'Option<KeyInfoResponse>',
+			},
+		},
+	},
+	types: {
+		MessageSourceId: 'u64',
+		KeyInfoResponse: {
+			msa_keys: 'Vec<AccountId>',
+			msa_id: 'MessageSourceId',
+		},
+	},
+};
```

Comment:
nit: not worth the dependency to import `@frequency-chain/api-augment`?

Reply:
I did this on a few tools. It limits the dependency on the Polkadotjs version trees for a cleaner setup. (Any you don't need the TS support)

Reply:
It keeps it simpler